dbld/images/packages/packages.txt

```diff
@@ -0,0 +1,171 @@
+#autogen
+[
+    autoconf                    [centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+    autoconf268                 [centos6]
```

Comment:
the names of distros/version numbers should be unified, I'd go for: <distro>-<version>, preferring code-names if those are widely used (e.g. ubuntu).

e.g. centos-6, centos-7, debian-stretch, debian-buster, ubuntu-xenial, ubuntu-trusty.